examples/rehastimp24_example.py

```diff
@@ -0,0 +1,170 @@
+from pyScienceMode2.channel import Channel, Point
+from pyScienceMode2.rehastim_interface import StimulatorP24 as St
```

Comment:
It must be two separate file : one rehastim2_interface the other rehastim_p24_interface and then you can import Rehastim2 or RehastimP24 class

Reply:
Done.